k8s/eks/yaml/multus-daemonset.yml

```diff
@@ -0,0 +1,289 @@
+# Note:
+#   This deployment file is designed for 'quickstart' of multus, easy installation to test it,
+#   hence this deployment yaml does not care about following things intentionally.
+#     - various configuration options
+#     - minor deployment scenario
+#     - upgrade/update/uninstall scenario
```

Comment:
If it's relevant, how do we handle these scenarios?

Reply:
Not our comment, left by the multus dev team.

Reply:
Yes, it says that the script we COPIED doesn't care about the listed scenarios. Do we care about them? Or why not?

Reply:
They do not seem relevant to our scope. If you wish, we can add a TODO to investigate all of these in the future.

Reply:
Do we have a story for how to upgrade multus for example?

Reply:
We do not, so as I said in the above comment, we can investigate the comments and also create a simple doc on multus upgrade.

Reply:
For now, please focus on the aspects of the scripts that would allow us to test it.

Reply:
(moving this into a follow-up issue)